tests/unit/test_routes.py

```diff
@@ -141,6 +141,8 @@ def add_policy(name, filename):
             "https://files.example.com/packages/{path}",
         ),
         pretend.call("rss.updates", "/rss/updates.xml", domain=warehouse),
+        pretend.call("rss.project_updates",
+                     "/rss/{name}/updates.xml", domain=warehouse),
```

Comment:
If you set the following:

```python
traverse="/{name}",
factory="warehouse.packaging.models:ProjectFactory",
```

you can expect a `project` argument to the `rss_project_updates` function, and a query will be unnecessary. This will also handle 404-ing if the project doesn't exist.